docs/pages/database-access/guides/postgres-cloudsql.mdx

```diff
@@ -0,0 +1,336 @@
+---
+title: Database Access with PostgreSQL on GCP Cloud SQL
+description: How to configure Teleport Database Access with GCP Cloud SQL PostgreSQL
+---
+
+# GCP Cloud SQL PostgreSQL
+
+## Enable Cloud SQL IAM Authentication
+
+Teleport uses [IAM database authentication](https://cloud.google.com/sql/docs/postgres/authentication)
+with Cloud SQL PostgreSQL instances.
+
+If you're [creating](https://console.cloud.google.com/sql/choose-instance-engine)
+a new PostgreSQL instance, make sure to add the `cloudsql.iam_authentication`
+database flag under "Customize your instance / Flags" section:
+
+![Enable IAM Authentication](../../../img/database-access/guides/cloudsql/iam-new-instance.png)
+
+To check whether IAM authentication is enabled for an existing Cloud SQL instance,
+look for the flag on the Configuration panel on the instance's Overview page:
+
+![Check IAM Authentication](../../../img/database-access/guides/cloudsql/iam-existing-instance.png)
+
+If it isn't enabled, you can add this flag using the "Edit configuration" dialog
+at the bottom of the Configuration panel. Note, changing this setting may
+require a database instance reboot.
+
+## Create Service Account for Database
+
+Teleport uses service accounts to connect to Cloud SQL databases.
+
+Go to the IAM & Admin [Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts)
+page and create a new service account:
+
+![Create Service Account](../../../img/database-access/guides/cloudsql/service-account-db.png)
+
+Press "Create".
+
+On the second step grant this service account the "Cloud SQL Instance User" role
+which will allow it to connect to Cloud SQL instances using IAM token for
+authentication:
+
+![Grant Cloud SQL Instance User to Service Account](../../../img/database-access/guides/cloudsql/service-account-db-grant.png)
+
+Press "Done".
+
+Now go back to the Users page of your Cloud SQL instance and add a new user
+account. In the sidebar, choose "Cloud IAM" authentication type and add the
+service account you've just created:
+
+![Add Cloud SQL User Account](../../../img/database-access/guides/cloudsql/add-user-account.png)
```

Comment:
Any reason for adding `teleport@` vs just `teleport`?

Reply:
We need to add service account as a database user, a service account name is an email.